src/tests.rs

```diff
@@ -27,23 +27,23 @@ pub fn get_cargo_workspace() -> &'static Path {
 
 pub fn walk_dir<'a>(
     root: &'_ Path,
-    skip: &'a [impl AsRef<OsStr>],
-    ext: impl for<'s> Fn(Option<&'s std::ffi::OsStr>) -> bool + 'static,
-) -> impl Iterator<Item = Result<walkdir::DirEntry, walkdir::Error>> + 'a {
-    walkdir::WalkDir::new(root)
-        .into_iter()
+    skip: &'static [impl AsRef<OsStr> + Send + Sync + 'a],
+    ext: impl for<'s> Fn(Option<&'s std::ffi::OsStr>) -> bool + Sync + Send + 'static,
+) -> impl Iterator<Item = Result<ignore::DirEntry, ignore::Error>> {
+    ignore::WalkBuilder::new(root)
```

Comment:
I believe this should use the [gitignore](https://docs.rs/ignore/0.4.18/ignore/gitignore/index.html) builder.

Reply:
We should probably use the global builder and also the `add` for `.git/info/exclude` if present, since we do probably want to check dotfiles as well (like the ones in `.changes`).

Reply:
Not needed, the builders defaults already does all that

https://docs.rs/ignore/0.4.18/ignore/struct.WalkBuilder.html#ignore-rules